readme.md

```diff
@@ -29,7 +29,7 @@ The plugin streamlines the entire configuration process and removes the need for
 - Jenkins 2.289.1+
 - Bitbucket Server 7.4+
 
-Note: Bitbucket Server 5.6 to 7.3 are also supported, but they're not recommended. This is because some plugin features are not available when using these versions. Instead, we recommend using Bitbucket Server 7.4+. With 7.0+ you can make use of pull request triggers for jobs. With 7.4+ you can set up an Application Link to have access to all plugin features.
+Note: Bitbucket Server 6.10 to 7.3 are also supported, but they're not recommended. This is because some plugin features are not available when using these versions. Instead, we recommend using Bitbucket Server 7.4+. With 7.0+ you can make use of pull request triggers for jobs. With 7.4+ you can set up an Application Link to have access to all plugin features.
```

Comment:
I wonder if we should note that 5.6 _may_ work it is just that we don't have any automated testing etc for it, and leave no guarantee for it. 5.5 (for example) simple does not work, as it does not support all the things the plugin relies on.

Thoughts?

Reply:
I am not too sure about providing versions that "may work" because we don't know when that will change with the future plugin changes. I would prefer keeping the documentation simple and aligned with what we test in our builds. Also it's easier to understand for future developers working on this plugin.